de-DE/docs/find.ndb

```diff
@@ -175,6 +175,7 @@ getlasterror()~Gibt den letzten von einem try-catch-Block verarbeiteten Fehler z
 getmatchingparens(STR)~Gibt die Position der passenen schlie�enden Klammer in STR zur�ck~klammer zu geschlossene geschweifte getmatchingparens() closing parenthesis bracket curly brace~getmatchingparens(
 getompthreads()~Gibt die Zahl der maximal m�glichen OpenMP-Threads zur�ck~CPU core logical logische kerne prozessor multithreading multiprocessing~getompthreads()
 getopt(STR,POS)~Gibt den Wert eines Parameters in der Zeichenkette STR, dessen Wert an der Stelle POS beginnt, als Zeichenkette zur�ck~parameter werte argumente options commandline values getopt()~getopt(
+getuilang()~Gibt Informationen zur Benutzersprache~version release candidate build datum date~getuilang()

```

Comment:
You should provide reasonable key words here. `version release candidate build datum date` does not reflect this. I'd recommend to use `language translation sprache deutsch german english englisch`